t04_refactored.py

```diff
@@ -245,9 +245,35 @@ def team_6_adv():
 
 
 def team_7_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1ksWilu6oBRWPoa5kZo2jyzkAOiAKqwtHYfMWGK7d27M/edit
+    Hira Karim
+    Claudia Pulido
+    :return:
+    """
+    direction = input("Go with him further [Up, or down]?")
```

Comment:
Here you ask the user for "Up" but the code looks for "up". Make them consistent.